release/release_test.go

```diff
@@ -41,10 +41,10 @@ func (s *ReleaseTestSuite) TestSetup(c *C) {
 	c.Check(release.Series, Equals, "16")
 }
 
-func makeMockLSBRelease(c *C) string {
+func makeMockOSRelease(c *C) string {
```

Comment:
and below, s/mockOS/os/, and let's make it call MockOSReleasePath by itself and return the restore function, so we don't need that everywhere.